Notebooks/DC2_Postage_Stamps.ipynb

```diff
@@ -61,37 +65,70 @@
     "zscale = ZScaleInterval()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## How To Make a Cutout Image\n",
+    "\n",
+    "The DM butler's `get` method enables postage stamp making, via a bounding box keyword argument. The functions below show how to make this bounding box, and apply it."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "def cutout_coadd_ra_dec(butler, ra, dec, filt='r', datasetType='deepCoadd', **kwargs):\n",
-    "    \"\"\"Produce a cutout from coadd from the given butler at the given RA, Dec in decimal degrees\n",
+    "def cutout_coadd_ra_dec(butler, ra, dec, filter='r', datasetType='deepCoadd', **kwargs):\n",
+    "    \"\"\"\n",
+    "    Produce a cutout from coadd from the given butler at the given RA, Dec in decimal degrees.\n",
+    "    \n",
+    "    Notes\n",
+    "    -----\n",
+    "    Trivial wrapper around 'cutout_coadd_spherepoint'\n",
     "    \n",
-    "    Trivial wrapper around 'cutout_from_coadd_spherepoint'\n",
+    "    Parameters\n",
```

Comment:
I wasn't sure about including these parameters in both functions, but decided in the end that good form is good form, and all dosctrings must follow it... (I did get lazier as the notebook went on though, I admit.) Merging these two cutout functions would help.

Reply:
I intentionally have them separate to illustrate wrapping a function that takes an annoying object (`afwGeom.SpherePoint`) with a function that takes what the user probably expects, which should be RA, Dec as`float`s.  Knowing that DM thinks in `SpherePoint`s is something this audience should be aware of.

The `SpherePoint` version is also useful when directly calling out from direct DM catalogs (which we don't do here, but would be one next extension).